js/lib/Signature.js

```diff
@@ -0,0 +1,29 @@
+const ethjsUtil = require('ethereumjs-util')
+
+class Signature {
+    constructor(v, r, s) {
+        Object.defineProperty(this, 'v', {
+            value: v,
+            writable: false
```

Comment:
i believe writable is `false` by default

Reply:
I kinda like the explicitness here